lxd/instance/drivers/driver_lxc.go

```diff
@@ -58,6 +58,29 @@ import (
 	"github.com/lxc/lxd/shared/units"
 )
 
+var idmappedMountsLock sync.RWMutex
```

Comment:
@brauner can these vars and functions be associated to the `type lxc struct ` type? They are only relevant for the lxc driver right? That way you also won't need to export them either.

Reply:
The problem is that this is a global feature not a container-local feature, i.e. the map indicates for all containers whether or not idmapping is supported for the associated backing fs.

Reply:
I think I'll move this into the storage driver part utils.go